### DISCUSSION TITLE
unclear error in the log trace when initializing the keyloak pod

### DISCUSSION THREAD

**Discussion by Idzana** [keycloak/keycloak#25701] [Archive]
Hi, I am using keylock operator (openshift) using high availability with multi-site fault tolerance.
However, at the last stage of setting up the external cache according to the official instructions (https://www.keycloak.org/high-availability/connect-keycloak-to-external-infinispan), I received an unclear error in log.  
For more information, I am attaching the CDR used and the configmap that caused the problem.
I’ll add on my own behalf that all variables and secrets are working.

[configure-kc-infini.txt](https://github.com/keycloak/keycloak/files/13715777/configure-kc-infini.txt)
[error-log.txt](https://github.com/keycloak/keycloak/files/13715778/error-log.txt)
[keycloak-test-keycloak.txt](https://github.com/keycloak/keycloak/files/13715779/keycloak-test-keycloak.txt)

**Comment by shawkins** [comment]
What version of the keycloak operator are you using? The most recent versions of the operator automaticially enable --verbose, which doesn't seem to be the case based upon your log. The guide doesn't indicate what if anything could be version specific - but it would be best to reproduce this on the latest release.

The important error log is:

```
java.lang.NullPointerException: Cannot invoke "org.infinispan.configuration.cache.ConfigurationBuilder.clustering()" because the return value of "java.util.Map.get(Object)" is null
```

This points to an issue with the configuration.

**Reply by Idzana** [reply]
> What version of the keycloak operator are you using?

Unfortunately, at now I can’t name the exact version (I'm not at work right now)... However, openshift downloaded the latest version that they had from the redhat store.

>  The guide doesn't indicate what if anything could be version specific - but it would be best to reproduce this on the latest release.

I have similar suspicions, and also that it is possible that the latest version of key-cloak may be incompatible with the latest version of datagrid (infinispan)

> This points to an issue with the configuration.

A puzzling problem, because I checked five times to see if I entered all the data correctly

**Reply by Idzana** [reply]
here is a versions
Keycloak operator: rhbk-operator.v22.0.7
Infinispan (Data Grid) operator: datagrid-operator.v8.4.12
